ProjectEuler/Problems/Problems_001_To_025.cs

```diff
@@ -141,11 +141,29 @@ private static bool IsDivisibleBy1Through20(int number)
         public static void Problem6()
         {
             Console.WriteLine("\n\n\n6. Find the difference between the sum of the squares of the first one hundred natural numbers and the square of the sum.");
+
+            long sumOfSquares = 0;
+
+            for(int i = 1; i <= 100; i++)
+            {
+                sumOfSquares += i * i;
+            }
+
+            long sumOfFirsthundredNums = 0;
+
+            for(int index = 1; index <= 100; index++)
+            {
+                sumOfFirsthundredNums += index;
```

Comment:
your variable for the sum of 100 numbers should have a capital 'H' for Hundred

Reply:
I missed that. I fixed it. Thanks!